modules/aws-in-a-box/0.0.42/MODULE.bazel

```diff
@@ -0,0 +1,30 @@
+module(
+    name = "aws-in-a-box",
+    version = "0.0.42",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_go", version = "0.41.0")
+
+go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(version = "1.21.0")
```

Comment:
This shouldn't be necessary as rules_go registers a default SDK. If you want a specific version, you can mark it with `dev_dependency`.